effectsize_exemplar_nonparametric.Rmd

```diff
@@ -0,0 +1,46 @@
+## Exemplar: Nonparametric effect size {#effectsize_exemplar_nonparametric}
+
+For a nonparametric test that produces a Z-score, like the Mann-Whitney U test or the Wilcoxon Signed-Rank test, an effect size can be computed as:
+
+<center>$r = \left|\frac{Z}{\sqrt{N}}\right|$</center>
+
+Above, Z is the Z-score and N is the number of observations in all groups [@Rosenthal1991a, p. 19). The result, *r*, is a variance-based effect size, like Pearson *r*, not a Cohen *d*-family effect size. The *r* can be squared to estimate the percentage of variance explained, however it will not be exactly equivalent to the Pearson *r*. 
+
+```
+TODO: This needs a domain where we can argue that the nonparametric approach is an exemplar analysis, then repeat structure of exemplar 1 with it
```

Comment:
This "incomplete parts" issue will happen in the future as well. We may have some impasse that will take long to resolve (not enough consensus, not enough knowledge/expertise, not knowing how to code them in R), even at the point after we merge the majority of the content to the `master` branch). Thus, I propose the following:

1. Explicitly mark the incompleteness in the file name, e.g., `wip_effectsize_exemplar_nonparametric.Rmd`.
2. We keep them included in the `_bookdown.yml` development branches (here `effect-size`) by default, so that contributors have an overview of its location in the exemplar and can link to them (e.g., from FAQ).
3. Before we merge each development to to the `master` branch. 
   3.1 We create separate issues for each of them, to make sure that they are tracked.
   3.2 We prefix each of these files with an "under construction" banner, with an explicit pointer to the associated issue created in 3.1. 

This will have two benefits: 
- Readers (authors and reviewers who use the guideline) know that they should "use this at their own risk".
- Readers (and potential authors) know the area that they can contribute, and can leave comments and know where/how to join in the discussion.